src/components/Blogposts.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import posts from '../data/posts.json';
+import {
+    Switch,
+    Route,
+    Link
+} from 'react-router-dom';
+import Blog
+    from "./Blog";
+
+
+export default function Blogposts({isAuthenticated}) {
+    const numberOfPosts = posts.length;
+
+    let listBlogposts = []
```

Comment:
hier zou ik de map methode gebruiken. hiermee doe je eigenlijk hetzelfde als wat je nu met de forloop hebt gedaan. alleen is de code een stuk korter